SECURITY.md

```diff
@@ -1,24 +1,59 @@
-contributions.
-
-Report security bugs by emailing the current owners of `raw-body`. This information
-can be found in the npm registry using the command `npm owner ls raw-body`.
-If unsure or unable to get the information from the above, open an issue
-in the [project issue tracker](https://github.com/stream-utils/raw-body/issues)
-asking for the current contact information.
-
-To ensure the timely response to your report, please ensure that the entirety
-of the report is contained within the email body and not solely behind a web
-link or an attachment.
-
-At least one owner will acknowledge your email within 48 hours, and will send a
-more detailed response within 48 hours indicating the next steps in handling
-your report. After the initial reply to your report, the owners will
-endeavor to keep you informed of the progress towards a fix and full
-announcement, and may ask for additional information or guidance.
+This document outlines security procedures and general policies for the raw-body
+project.
+
+  * [Reporting a Bug or Security Vulnerability](#reporting-a-bug-or-security-vulnerability)
+  * [Disclosure Policy](#disclosure-policy)
+  * [Comments on this Policy](#comments-on-this-policy)
+
+## Reporting a Bug or Security Vulnerability  
+
+The `raw-body` team and community take all security vulnerabilities seriously. 
+Thank you for improving the security of raw-body and related projects. 
+We appreciate your efforts in responsible disclosure and will make every effort 
+to acknowledge your contributions.  
+
+After the initial response to your report, the owners commit to keeping you informed
+about the progress toward a fix and the final announcement, and they may request additional
+information or clarification during the process.
+
+### Reporting Security Bugs via GitHub Security Advisory 
+
+The preferred way to report security vulnerabilities is through 
+[GitHub Security Advisories](https://github.com/advisories). 
+This allows us to collaborate on a fix while maintaining the 
+confidentiality of the report.  
+
+To report a vulnerability
+([docs](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability)):  
+1. Visit the **Security** tab of the affected repository on GitHub.  
+2. Click **Report a vulnerability** and follow the provided steps.  
+
+### Reporting via Email  
+
+Report security bugs by emailing the current owners of `raw-body`. This information can be found in the npm registry using the command `npm owner ls raw-body`
+
+To ensure a timely response, please include all relevant details directly in the email body rather than linking to external sources or attaching files.  
+
+The lead maintainer will acknowledge your email within 72 hours and provide an initial response outlining the next steps. The security team will keep you updated on the progress and may request additional details.  
```

Comment:
I will suggest to avoid email reporting, unless you have a single email address that can notify all the maintainers at once. Otherwise is adding an additional step into the security triage efforts.